src/governance.cpp

```diff
@@ -147,6 +147,11 @@ void CGovernanceManager::ProcessMessage(CNode* pfrom, std::string& strCommand, C
         uint256 nHash = govobj.GetHash();
         std::string strHash = nHash.ToString();
 
+        LOCK2(cs_main, cs);
+
+        pfrom->setAskFor.erase(nHash);
+        mapAlreadyAskedFor.erase(nHash);
```

Comment:
Does it make sense to clear mapAlreadyAskedFor ?

This is a limited_map so it's size is bounded.

If we clear it here I guess it means we could rerequest this item again now, but why would we want to do that if we've already received it ?